drivers/afe/ad4110/ad4110.c

```diff
@@ -830,6 +966,26 @@ int32_t ad4110_setup(struct ad4110_dev **device,
 	if (ret)
 		goto err_spi;
 
+	ret = ad4110_set_bipolar(dev, init_param.bipolar);
+	if (ret)
+		goto err_spi;
+	dev->bipolar = init_param.bipolar;
```

Comment:
why not making the assignment `dev->bipolar=init_param.bipolar;` inside the setter function if the write is completed successfully?
Same applies for all the other setter functions.

In this manner the users can benefit of these functions outside the driver and the devices structure is updated accordingly.

Reply:
Moved the device descriptor update inside the setter function.